model/map/src/main/java/org/keycloak/models/map/client/MapClientEntity.java

```diff
@@ -415,7 +415,7 @@ public void removeAuthenticationFlowBindingOverride(String binding) {
     }
 
     public void setAuthenticationFlowBindingOverride(String binding, String flowId) {
-        this.updated |= ! Objects.equals(this.nodeReRegistrationTimeout, nodeReRegistrationTimeout);
+        this.updated |= Objects.equals(this.nodeReRegistrationTimeout, nodeReRegistrationTimeout);
```

Comment:
This one fixes org.keycloak.testsuite.cli.KcinitTest.testBrowserContinueAuthenticator

Reply:
Isn't this line (even in the original) utterly wrong? The `equals` should be on the parameters, not on nodeReRegistrationTimeout